### PR TITLE
Initial ODK import

### PR DESCRIPTION
**Pull request by peritus**

### CODE REVIEW

**Comment by readevalprint** [comment]
+1

File: gather2_core/settings.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Base(Configuration):
         'rest_framework',
         'django_extensions',
         'gather2_core',
+        'gather2_odk_import',
```
Comment (readevalprint) [inline gather2_core/settings.py]: Yeah this is what I was mentioning, these need to be completely separate and should interface over http. This is why I objected to removing the `-` in the path to reduce the urge to import it.

File: gather2_core/urls.py [hunk 2]
```diff
@@ -35,5 +36,6 @@
     url(r'^api-auth/', include('rest_framework.urls',
                                namespace='rest_framework')),
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^odk/', include(gather2_odk_import.urls)),
```
Comment (readevalprint) [inline gather2_core/urls.py]: Core should not be aware of any importers under any circumstance.

File: gather2_core/settings.py (hunk 1 above)
Comment (peritus) [inline gather2_core/settings.py]: At this point they are two separate python modules or "django apps", but can be installed together in the same "django project" of which this settings.py is an example. They don't interact (yet).

File: gather2_core/urls.py (hunk 2 above)
Comment (peritus) [inline gather2_core/urls.py]: Guess we need to talk about goals and architecture then.

**Comment by readevalprint** [comment]
Hey @peritus This PR has an explicit -1 so definitely was not approved. So why did you merge this in after we discussed the goals and architecture last week and you even confirmed that they would not share code like this PR is implementing?

**Pull request merged**